src/iiif/tools.py

```diff
@@ -37,15 +42,25 @@ def create_cantaloupe_url_and_headers(request_meta, url_info, iiif_url, metadata
         headers['X-Forwarded-Host'] = request_meta['HTTP_X_FORWARDED_HOST']
 
     if url_info['source'] == 'edepot':
-        return iiif_url, headers
+        iiif_image_url = f"{settings.IIIF_BASE_URL}:{settings.IIIF_PORT}/iiif/{iiif_url}"
```

Comment:
ik zou zelf deze logic naar functies verplaatsen, zoals get_edepot_url(), get_wabo_url(), etc.

Reply:
Goed punt. Ga ik in een volgend issue oplossen.